invenio_app_rdm/utils/files.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Files utilities."""
+
+from __future__ import absolute_import, print_function
+
+from datetime import datetime
+from urllib.parse import quote
+
+import sqlalchemy as sa
+from flask import current_app
+from flask_mail import Message
+from invenio_files_rest.models import FileInstance
+from invenio_rdm_records.records.models import RDMDraftMetadata, RDMRecordMetadata
+
+
+def checksum_verification_files_query():
+    """Return a FileInstance query taking into account file URI prefixes."""
+    files = FileInstance.query
+    uri_prefixes = current_app.config.get(
+        "FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES"
+    )
+    if uri_prefixes:
+        files = files.filter(
+            sa.or_(*[FileInstance.uri.startswith(p) for p in uri_prefixes])
+        )
+    return files
+
+
+URLS = {
+    "deposit_html": "{base}/uploads/{id}",
+    "record_html": "{base}/records/{id}",
+}
+
+
+def ui_link_for(tpl, **kwargs):
+    """Create an UI link using specific template."""
+    UI_URL = current_app.config.get("SITE_UI_URL")
```

Comment:
Is `SITE_UI_URL` the correct config to be used here? It's meant to be the site ui url so e.g. `https://zenodo-rdm.web.cern.ch/records/123456` is parsed for a faulty record and sent in the report.